File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1158.cs
using System;

class URI
{
    static void Main() {
        int n = int.Parse(System.Console.ReadLine().Trim()), i, j, x, y, soma;
        for(i=0;i<n;i++){
            string nums = System.Console.ReadLine().Trim();
            string[] ns = nums.Split(' ');
            x = int.Parse(ns[0]);
            y = int.Parse(ns[1]);
            if(x%2==0){
                x++;
            }
            soma=0;
            for(j=0;j<y;j++){
                soma+=x;
                x+=2;
            }
            Console.Write("{0}\n", soma);
        }
    }
}

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1158.rb
n = gets.to_i

for i in 1..n do
    ns = gets.chomp().split(' ')
    x = ns[0].to_i
    y = ns[1].to_i
    if(x%2==0) then
        x=x+1
    end
    soma=0
    for j in 1..y do
        soma=soma+x
        x=x+2
    end
    puts(soma)
end